From b2e0a7f077d3b803981fae4977941ea4d6085fe5 Mon Sep 17 00:00:00 2001
From: Steve Dickson <SteveD@redhat.com>
Date: Thu, 26 Jun 2014 15:54:34 -0400
Subject: [fs] nfs: Comment is now wrong and redundant to code

Message-id: <1403798077-28799-76-git-send-email-steved@redhat.com>
Patchwork-id: 84920
O-Subject: [RHEL7.1 PATCH 75/78] NFSv4.1: Comment is now wrong and redundant to code
Bugzilla: 1113248
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>

From: Tom Haynes <Thomas.Haynes@primarydata.com>

The save of the write offset was removed some time ago, so that
part of the comment is bogus.

The remainder is pretty self-evident.

So off with it!

Signed-off-by: Tom Haynes <Thomas.Haynes@primarydata.com>
Signed-off-by: Trond Myklebust <trond.myklebust@primarydata.com>
Upstream Id:
Bugzilla: 1113248
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfs/filelayout/filelayout.c b/fs/nfs/filelayout/filelayout.c
index a008e85..c7fbc91 100644
--- a/fs/nfs/filelayout/filelayout.c
+++ b/fs/nfs/filelayout/filelayout.c
@@ -608,10 +608,7 @@ filelayout_write_pagelist(struct nfs_pgio_data *data, int sync)
  fh = nfs4_fl_select_ds_fh(lseg, j);
  if (fh)
   data->args.fh = fh;
- /*
-  * Get the file offset on the dserver. Set the write offset to
-  * this offset and save the original offset.
-  */
+
  data->args.offset = filelayout_get_dserver_offset(lseg, offset);
 
  /* Perform an asynchronous write */
-- 
1.7.1